lib/influxdb/rails/middleware/request_subscriber.rb

```diff
@@ -19,6 +19,7 @@ def tags(payload)
             status:      payload[:status],
             format:      payload[:format],
             http_method: payload[:method],
+            exception:   payload[:exception].try(:first),
```

Comment:
[nitpick] Safe navigation would work here.

```suggestion
            exception:   payload[:exception]&.first,
```

Reply:
pushed